app/views/feedback/new.html.haml

```diff
@@ -0,0 +1,15 @@
+
+%section#content.content-main.form_wrapper{role: "main"}
+  %h1 Your Feedback
+  = simple_form_for(@form, url: feedback_path) do |f|
+    = f.input :comment, as: :text
+    = f.hidden_field :url
+    %h3 About You
```

Comment:
Maybe it's not a heading at all. Is it a fieldset with a legend? Like in `nodes/edit.html.haml`?